internal/app/pactproxy/proxy.go

```diff
@@ -247,9 +250,9 @@ func (a *api) interactionsWaitHandler(c echo.Context) error {
 
 func (a *api) indexHandler(c echo.Context) error {
 	req := c.Request()
-	log.Infof("proxying %s %s", req.Method, req.URL.Path)
```

Comment:
Did you mean to change the log line, or is this just for debugging?

Reply:
It helped me a lot while debugging, when I had no idea that swagger autogenerated code does not allow to change `Content-Type`, even when provided via parameters (defaulted to `application/json`). I can revert this change, but in my opinion, it might be helpful in the future